alphalens/performance.py

```diff
@@ -425,3 +426,123 @@ def average_cumulative_return(q_fact):
         return pd.DataFrame( {'mean': q_returns.mean(axis=1), 'std': q_returns.std(axis=1)} ).T
 
     return quantized_factor.groupby(quantized_factor).apply(average_cumulative_return)
+
+
+def get_vifs(data):
+    '''
+    Computes variance inflation factors (VIFs) for a set of random variables.
+    For more information, see
+    https://en.wikipedia.org/wiki/Variance_inflation_factor
+
+    Parameters
+    ----------
+    data : pd.DataFrame
+        DataFrame with observations of random variables.
+        - Columns are observations of a random variable, and each row makes up
+        one observation for each random variable.
+    '''
+    vifs = np.zeros(len(data.columns))
+    for i in range(len(data.columns)):
+        r2 = OLS(data.iloc[:, i],
```

Comment:
actually, the extra indents for `.fit()` and `.rsquared` seem to violate pep8... I've added the parentheses but the pep8 suggests having all lines on the same indentation level